docs/cpu-specific-build-macros.md

```diff
@@ -60,6 +60,12 @@ For Cortex-A53, following errata build flags are defined :
      CPU. This needs to be enabled only for revision <= r0p3 of the CPU. From
      r0p4 and onwards, this errata is enabled by default.
 
+*   `ERRATA_A53_855873`: This applies errata 855873 workaround to Cortex-A53
+     CPUs. Though the erratum is present in every revision of the CPU,
+     this workaround is only applied to CPUs from r0p3 onwards, which feature
+     a chicken bit in CPUACTLR_EL1 to enable a hardware workaround.
+     Earlier revisions of the CPU are meant to be patched by the rich OS.
```

Comment:
"Rich OS" is the generic term for a general purpose OS like Linux, as opposed to a secure OS which is meant to be lean to have a smaller attack surface.
Originally I had "Linux" in there, but was asked to change this to "rich OS" to be more generic (as it could be *BSD as well, for instance).